src/connectors/pocketcasts.js

```diff
@@ -10,7 +8,7 @@ Connector.currentTimeSelector = '.current-time';
 Connector.remainingTimeSelector = '.time-remaining';
 Connector.trackArtSelector = '.player-image img';
 
-let origRemainingSelector = Connector.getRemainingTime;
+const origRemainingSelector = Connector.getRemainingTime;
```

Comment:
Can we have `origGetRemainingTime` instead? This is not a selector, but a function, so the variable name might be confusing.

Reply:
Good catch